modules/querying/partials/histogram-api.adoc

```diff
@@ -0,0 +1,130 @@
+[#_compute_histogram_statistics]
+=== Compute histogram statistics
+
+`POST :14240/gsqlserver/gsql/stats/histogram`
+
+This endpoint computes histograms for a specified attribute of a vertex type.
```

Comment:
Similarly here, can you call out that this is a preview feature and subject to change.